lib/blocs/guest-repository.dart

```diff
@@ -200,6 +201,26 @@ class GuestRepository implements GuestRepositoryInterface {
     }
   }
 
+  @override
+  Future<Guest> signWaiver(Guest owner) async {
+    try {
+      var body = jsonEncode(owner.toJson());
+      final response = await http.post('http://localhost:7777/users/waiver',
```

Comment:
we should make this changeable from within the app... so if we have pi issues or IP issues on site we're not fucked.